src/app/space/create/deployments/components/deployment-card/deployment-card.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
-    <span class="pficon pficon-ok"></span>
+    <span class="pficon pficon-ok" data-toggle="tooltip" data-placement="top" title="Everything looks okay"></span>
```

Comment:
When I apply this locally, the tooltip is not placed at the "top" and doesn't look like the tooltip as shown in the design mockup.

The patternfly docs point to the bootstrap docs: https://getbootstrap.com/docs/3.3/javascript/#tooltips

which says this: `For performance reasons, the Tooltip and Popover data-apis are opt-in, meaning you must initialize them yourself.`

I suspect that the tooltip data-api not being initialized is why I'm seeing it presented visually differently than the patternfly standard tooltip.